app/multitenant/aws_collector.go

```diff
@@ -368,7 +381,12 @@ func (c *awsCollector) Add(ctx context.Context, rep report.Report, buf []byte) e
 		return err
 	}
 
-	reportSize, err := c.s3.StoreReportBytes(ctx, reportKey, buf)
+	var reportSize int
+	err = withBackoff(func() error {
+		var err error
+		reportSize, err = c.s3.StoreReportBytes(ctx, reportKey, buf)
```

Comment:
I don't see why we would want to retry S3. If there are specific retry-able errors (are there?) then maybe, but not in general.